examples/contrib/gp/sv-dkl.py

```diff
@@ -70,6 +70,7 @@ def train(args, train_loader, gpmodule, optimizer, loss_fn, epoch):
         loss = loss_fn(gpmodule.model, gpmodule.guide)
         loss.backward()
         optimizer.step()
+        batch_idx = batch_idx + 1
```

Comment:
We report the number of training data has been trained in the epoch at `batch_idx`, so for `batch_idx = 0`, we should report `len(batch_data)` instead of `0`.